velox/common/caching/tests/AsyncDataCacheTest.cpp

```diff
@@ -89,7 +89,10 @@ class AsyncDataCacheTest : public testing::Test {
     }
   }
 
-  void initializeCache(uint64_t maxBytes, int64_t ssdBytes = 0) {
+  void initializeCache(
+      uint64_t maxBytes,
+      int64_t ssdBytes = 0,
+      bool multiCachePath = false) {
```

Comment:
Is it better to specify the number of cache paths?

Reply:
Hm, this is mainly used to test multiple cache directories, but it is ok to me that changing it to number to facilitate subsequent related tests.